pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -569,15 +612,22 @@ func (l *listUsersQuery) expandExclusion(
 		close(subtractFoundUsersCh)
 	}()
 
-	baseFoundUsersMap := make(map[string]struct{}, 0)
+	baseFoundUsersMap := make(map[string]foundUser, 0)
 	for fu := range baseFoundUsersCh {
-		key := tuple.UserProtoToString(fu)
-		baseFoundUsersMap[key] = struct{}{}
+		key := tuple.UserProtoToString(fu.user)
+		baseFoundUsersMap[key] = fu
 	}
-	subtractFoundUsersMap := make(map[string]struct{}, len(baseFoundUsersMap))
+
+	subtractFoundUsersMap := make(map[string]foundUser, len(baseFoundUsersMap))
+	negatedSubtractionMap := make(map[string]struct{})
```

Comment:
`negatedSubtractionMap` doesn't appear to be used anymore besides being assign to below. But its never being read from.

Reply:
Removed